core/crypto.mk

```diff
@@ -6,6 +6,13 @@ CFG_CRYPTO_SIZE_OPTIMIZATION ?= y
 
 ifeq (y,$(CFG_CRYPTO))
 
+###############################################################
+# Platform independent crypto-driver configuration
```

Comment:
In the commit message, you say that the platform crypto configuration should have higher priority than the generic stuff, yet we talk about platform-independent stuff here. I can't make sense of it.

Reply:
Would that make more sense?
```
###############################################################
# Platform crypto-driver configuration. It has a higher priority over the
# generic crypto configuration below.
###############################################################
```

Reply:
Yes, thanks.